app/views/pages/dashboard.html.haml

```diff
@@ -0,0 +1,5 @@
+%section.container
+%h1= "Dashboard"
+%p
+  %li= link_to t('.menu',day: 'Test Day'), menu_path(1) , class: "btn btn-primary btn-sm mt-10"
```

Comment:
menu_path(1) ???

please review your spaces

Reply:
It's just a test because I don't know how to do menus properly